front/src/applications/stdcmV2/components/StdcmOpSchedule.tsx

```diff
@@ -95,9 +96,8 @@ const StdcmOpSchedule = ({
               onArrivalTypeChange(e as ArrivalTimeTypes);
             }
           }}
-          options={['preciseTime', 'asSoonAsPossible']}
+          {...createStringSelectOptions(['preciseTime', 'asSoonAsPossible'])}
```

Comment:
Wasn't the point of using `createStringSelectOptions` not to have to pass `getOptionLabel` ?

Reply:
Yes but it's a special case where we have a translation as label don't think that we want a function for each case ?

Reply:
Hmm ok :)